packages/wallets/src/components/MT5CreatePassword/MT5CreatePassword.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import MT5PasswordIcon from '../../public/images/ic-mt5-password.svg';
 import PasswordShowIcon from '../../public/images/ic-password-show.svg';
+import { useModal } from '../ModalProvider';
+import WalletModal from '../WalletModal';
 
 const MT5CreatePassword = () => {
+    const { hide } = useModal();
+
     return (
-        <div className='wallets-create-password'>
+        <WalletModal className='wallets-create-password' should_show_close_icon onClickCloseIcon={hide}>
```

Comment:
```suggestion
        <WalletModal className='wallets-create-password' has_close_icon onClickCloseIcon={hide}>
```
Maybe a smaller name for props just a suggestion
Ignore it

Reply:
ooh that's a better name, will do that